packages/zod/src/schemas.ts

```diff
@@ -0,0 +1,107 @@
+import { z } from "zod";
+
+function isUndefined(value: unknown): boolean {
+  return typeof value === "undefined";
+}
+
+function isUnset(value: unknown): boolean {
+  return (typeof value === "string" && !value.length) || isUndefined(value);
+}
+
+function defaultTransformer(defaultValue: unknown) {
+  return (arg: unknown) => {
+    return isUnset(arg) ? defaultValue : arg;
+  };
+}
+
+export function booleanSchema() {
+  const schema: z.ZodType = z.string();
+
+  return schema.transform((arg, ctx): boolean => {
+    const varName = ctx.path[0];
+    const arg_ = arg.toLowerCase();
+
+    if (arg_ !== "true" && arg_ !== "false") {
+      ctx.addIssue({
+        code: z.ZodIssueCode.custom,
+        message: `${varName} must be either true or false`,
+      });
+
+      return z.NEVER;
+    }
+
+    return arg_ === "true";
```

Comment:
If we want to prevent the blob storages be set to `true` if the ENV were kept empty. Should not the default value be `false` here?

Reply:
If we use `"false"` here, ,then we would end up with the opposite result instead.

For example: say we set one of the enabled env vars to "true" (`arg_="true"`) then this zod transform fn would return false as `"true" === "false"` is false

Reply:
Right, I miss read it.